proxy/core/src/graal/reflect-config.json

```diff
@@ -1225,8 +1225,8 @@
 {
   "name":"sun.nio.ch.SelectorImpl",
   "fields":[
-    {"name":"publicSelectedKeys"}, 
-    {"name":"selectedKeys"}
+    {"name":"publicSelectedKeys", "allowUnsafeAccess": true},
+    {"name":"selectedKeys", "allowUnsafeAccess": true}
```

Comment:
@dwijnand I believe this needs to be addressed in proxy/cassandra and proxy/postgres as well?